### ISSUE TITLE
libgdx always append to proguard file every time running project on iOS-moe

### ISSUE THREAD

**Issue by haxpor** [libgdx/libgdx#4821]
Please ensure you have given all the following requested information in your report.

#### Issue details
Every time you hit run from Android Studio for iOS-MOE, it will write the same content that already there in `proguard.append.cfg`.

These two lines

```
-keep class com.badlogic.** { *; }
 -keep enum com.badlogic.** { *; }
```

I think it must be something about `build.gradle` file for iOS-MOE.

#### Reproduction steps/code
Run any project on iOS-moe, and take a look at `proguard.append.cfg` file.

#### Version of LibGDX and/or relevant dependencies
1.9.6
Version of MOE plugin is not relevant, but it happens to me on both 1.3.1, and 1.3.2.

#### Stacktrace
Not applicable

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [x] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by Leejjon** [comment]
I developed and released my app with libgdx 1.9.6 on the App Store and never had this problem. For iOS you have to use the new 1.4.0 version now anyway:
https://discuss.multi-os-engine.org/t/moe-1-4-0-with-xcode-9-support-is-available-for-testing/1224/16

I'd recommend updating to all the latest versions and see if your problem is still there.

**Comment by crykn** [comment]
Closing this, as iOS moe is discontinued and the last comment on this issue was posted three years ago.
